oximeter/db/src/lib.rs

```diff
@@ -35,11 +41,12 @@ pub mod model;
 #[cfg(any(feature = "oxql", test))]
 pub mod oxql;
 pub mod query;
-#[cfg(any(feature = "oxql", feature = "sql", test))]
+#[cfg(any(feature = "oxql", feature = "sql", feature = "oxdb", test))]
```

Comment:
Is this mod needed on the `oxdb` feature? It looks like you've moved the `make_client()` method, and I don't see any others called unless the `oxql` or `sql` is also enabled.

Reply:
yep, just leftover.

Reply:
making a few more minor updates around oxdb and this, coming up.